src/auth/defraId-client.js

```diff
@@ -0,0 +1,49 @@
+import Wreck from '@hapi/wreck'
```

Comment:
let's rename this file to matching the same hyphenated convention as the other files 
`/defra-id-client.js`